src/system/components/handler.clj

```diff
@@ -1,7 +1,8 @@
 (ns system.components.handler
   (:require [com.stuartsierra.component :as component]
             [lang-utils.core :refer [contains+? ∘]]
-            [compojure.core :as compojure]))
+            [compojure.core :as compojure]
+            [bidi.ring :as bidi]))
```

Comment:
Conversely, this requires people who only use bidi to have compojure as a dependency. Shouldn't compojure also by resolved dynamically?